styles/tab-inventory.css

```diff
@@ -59,15 +97,155 @@ table th {
 }
 
 .item-img-wrapper .item-show:hover {
-  background: rgba(0, 0, 0, 0.3);
+    background: rgba(0, 0, 0, 0.3);
 }
 
 .item-img-wrapper .item-show .fa {
-  display: none;
-  color: rgb(200, 200, 200);
-  font-size: 28px;
+    display: none;
+    color: rgb(200, 200, 200);
+    font-size: 28px;
 }
 
 .item-img-wrapper .item-show:hover .fa {
-  display: inline-block;
+    display: inline-block;
+}
+
+.weapon-header {
```

Comment:
So much weapon stuff. I think it makes sense to create a new css file for weapon display classes